Monika After Story/game/script-affection.rpy

```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
 
         ## DISTRESSED quips
         quips = [
+            _("..."),
             _("...Yes?"),
-            _("...Oh?"),
-            _("...Huh?"),
             _("...Hm?"),
-            _("We can try talking, I guess."),
-            _("I guess we can talk."),
-            _("Oh... You want to talk?"),
-            _("If you want to talk, go ahead."),
-            _("We can talk if you really want to."),
+            _("...Hello?"),
```

Comment:
I'm not sure how much this works, these bits aren't greetings, and she'll have already greeted you having launched MAS.
```suggestion
```